src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -94,15 +94,18 @@ export function VisualizeListingController($injector, createNewVis) {
       });
   };
 
-  this.deleteSelectedItems = function deleteSelectedItems(selectedIds) {
-    return visualizationService.delete(selectedIds)
-      .catch(error => {
-        toastNotifications.addError(error, {
-          title: i18n.translate('kbn.visualize.visualizeListingDeleteErrorTitle', {
-            defaultMessage: 'Error deleting visualization',
-          }),
-        });
+  this.deleteSelectedItems = function deleteSelectedItems(selectedItems) {
+    return Promise.all(
+      selectedItems.map(item => {
+        return savedObjectClient.delete(item.savedObjectType, item.id);
```

Comment:
I already commented on that in the old PR:
The "old" delete function also called chrome.untrackNavLinksForDeletedSavedObjects in src/legacy/ui/public/saved_objects/saved_object_loader.js line 71, we probably also have to do that, even though I'm not sure what it does.

Is it solved somewhere else or got it lost during the transfer?

Reply:
I just added it to the master branch PR, so it'll come in there. I won't bother cherry-picking it back to this.